sky/utils/schemas.py

```diff
@@ -659,8 +659,27 @@ def get_service_schema():
                         'minimum': 0,
                     },
                     'target_qps_per_replica': {
-                        'type': 'number',
-                        'minimum': 0,
+                        'anyOf': [
+                            {
+                                'type': 'number',
+                                'minimum': 0,
+                            },
+                            {
+                                'type': 'object',
+                                'patternProperties': {
+                                    # Pattern for accelerator types like
+                                    # "H100:1", "A100:1"
+                                    '^[A-Z0-9]+:[0-9]+$': {
+                                        'type': 'number',
+                                        'minimum': 0,
+                                    }
+                                },
+                                'additionalProperties': False,
+                            }
+                        ]
+                    },
+                    'use_instance_type_aware': {
+                        'type': 'boolean',
```

Comment:
Also, this option is quite redundant to me. IIUC, `target_qps_per_replica` always co-exists with this config, so why not assuming using instance type awared mode when any of `target_qps_per_replica` is given.